servers/physics_2d/collision_object_2d_sw.cpp

```diff
@@ -73,6 +73,29 @@ void CollisionObject2DSW::set_shape_transform(int p_index, const Transform2D &p_
 	_shapes_changed();
 }
 
+void CollisionObject2DSW::set_shape_as_disabled(int p_idx, bool p_disabled) {
+	ERR_FAIL_INDEX(p_idx, shapes.size());
+
+	CollisionObject2DSW::Shape &shape = shapes[p_idx];
+	if (shape.disabled == p_disabled)
+		return;
+
+	shape.disabled = p_disabled;
+
+	if (!space)
+		return;
+
+	if (p_disabled && shape.bpid != 0) {
+		space->get_broadphase()->remove(shape.bpid);
+		shape.bpid = 0;
+		_update_shapes();
+	}
+	else if (!p_disabled && shape.bpid == 0) {
```

Comment:
There's a style issue here: https://travis-ci.org/godotengine/godot/jobs/347916013

Could you amend the commit to fix it? See also http://docs.godotengine.org/en/latest/community/contributing/code_style_guidelines.html on how to setup code style checks locally.

Reply:
Working on it! :)

Reply:
Fixed code style and rebased. Also installed the ClangFormat plugin for Visual Studio, so this should not become an issue again in future ... hopefully :smile: